driver-async/src/test/functional/com/mongodb/async/client/RetryableWritesTest.java

```diff
@@ -115,9 +116,11 @@ public void setUp() {
         }
         collectionHelper = new CollectionHelper<Document>(new DocumentCodec(), new MongoNamespace(databaseName, collectionName));
         BsonDocument clientOptions = definition.getDocument("clientOptions", new BsonDocument());
-        mongoClient = MongoClients.create(getMongoClientBuilderFromConnectionString()
-                .retryWrites(clientOptions.getBoolean("retryWrites", BsonBoolean.FALSE).getValue())
-                .build());
+        MongoClientSettings.Builder builder = getMongoClientBuilderFromConnectionString();
+        if (clientOptions.containsKey("retryWrites")) {
+            builder.retryWrites(clientOptions.getBoolean("retryWrites").getValue());
```

Comment:
Since the property is of type `boolean` and not `Boolean`, we have to avoid changing the default unless clientOptions actually contains the `retryWrites` key.